src/components/Section.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import { Add } from '~/assets/image';
+import * as storage from '~/storage/LocalStorage';
+import { FormContext } from '~/components/FormContext';
+import EmptyCard from '~/components/emptycard/EmptyCard';
+import CardHeader from '~/components/cardheader/CardHeader';
+import CardFooter from '~/components/cardfooter/CardFooter';
+import { capitalize, camelToTitle, singularize } from '~/utilities/string';
+
+const Section = ({ dataKey, component, children }) => {
```

Comment:
passed the dataKey prop so we can reuse the component and get the context data like `const contextData = context.data.get[dataKey];`